src/lib/init.js

```diff
@@ -49,10 +49,25 @@ export function init(configUpdates) {
 
 				// set cookies on digitrust domain after consent submitted
 				const { addEventListener, getVendorConsents } = cmp.commands;
+				if (config.digitrust.pixel) {
+					addEventListener('consentStringUpdated', digitrustPixel);
+				}
 				if (config.digitrust.redirects) {
 					addEventListener('consentStringUpdated', digitrustRedirect);
 				}
 
+				function digitrustPixel() {
+					getVendorConsents([64], (result) => {
```

Comment:
How about moving 64 to a constant? It's used several times in this file.

Reply:
moved into metadata.json